app/javascript/packages/polyfill/index.js

```diff
@@ -1,8 +1,7 @@
-import 'promise-polyfill/src/polyfill';
```

Comment:
`core-js` includes a `Promise` and `URL` polyfill, so we don't need the other one. Which is nice because it lets us use fewer dependencies.